client/my-sites/customer-home/cards/features/stats/index.jsx

```diff
@@ -70,11 +70,17 @@ export const StatsV2 = ( {
 		}
 	}, [ isSiteUnlaunched ] );
 
+	const newSiteCopy =
+		[ 'en', 'en-gb' ].includes( i18nCalypso.getLocaleSlug() ) ||
+		i18nCalypso.hasTranslation( "We're collecting your stats! Come back and see after a week." )
+			? translate( "We're collecting your stats! Come back and see after a week." )
```

Comment:
Per the [slack convo](p1620681683252000/1620626375.227200-slack-C01VA0R5NKY), let's change this to “No stats to display yet. Publish or share a post to get some traffic to your site.”  Or if you felt strongly about another iteration from the slack convo, that's fine too.   (All of the suggestions in the slack convo are an improvement over the "Come back and see....").   Then we can ship this. :)

Reply:
Pushed this suggestion 👍 
Longer message means wrapping, but I think I've handled it nicely (see updated screenshot in description).